vignettes/geological-completeness.Rmd

```diff
@@ -246,7 +246,11 @@ ggplot(counts, aes(fill = environment, y = count, x = mid_ma)) +
         legend.position = "top") +
   # Add geological timescale
   coord_geo()
```

Comment:
Should we use `coord_geo()` with `time_bins(rank = "period")` (with the column names changed) to match the start and end of the time intervals from `bins` that's used above? For example, there's a little bit of the Ediacaran currently showing because the Cambrian only goes to 538.8 now (according to the latest ICS).

Reply:
Instead of adding this in here, I've changed the previous code (L187–197) to use the stage-level bins from Macrostrat for time binning (this seemed more appropriate as presumably some/most of the unit ages are being pulled from the latest ICS).